ui/app/com/gu/recipeasy/models/CuratedRecipeForm.scala

```diff
@@ -0,0 +1,51 @@
+package models
+
+import com.gu.recipeasy.models._
+import TagHelper._
+import java.time.OffsetDateTime
+import automagic._
+
+case class CuratedRecipeForm(
+  id: String,
+  title: String,
+  body: String,
+  serves: Option[Serves],
+  ingredientsLists: Seq[DetailedIngredientsList],
+  articleId: String,
+  credit: Option[String],
+  publicationDate: String,
+  status: String,
+  times: TimesInMins,
+  steps: Seq[String],
+  tags: FormTags
+)
+
+object CuratedRecipeForm {
+
+  def toForm(r: CuratedRecipe): CuratedRecipeForm = {
```

Comment:
I thought you said CuratedRecipe was not common? If it is common, then ignore me.